### DISCUSSION TITLE
Facing issues with Groq

### DISCUSSION THREAD

**Discussion by himanshud2611** [microsoft/autogen#4342] [Q&A] [answered]
I'm using Groq but it is generating this error
"ImportError: Please install `groq` to use the Groq API."

![image](https://github.com/user-attachments/assets/d88ada7d-107e-4bbb-8111-5e9ba56375c5)

**Comment by jackgerrits** [comment, marked as answer]
Did you install AutoGen with the `groq` extra enabled?

```sh
pip install autogen-agentchat[groq]~=0.2
```

**Comment by himanshud2611** [comment]
Thanks! It worked.
